### PR TITLE
TW 538 Add note to token scanner page

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/api-security/token-scanner.md [hunk 1]
```diff
@@ -40,23 +40,25 @@ The Postman Token Scanner scans your public workspaces, collections, environment
...
 * Change the workspace visibility to Public.
 * Share a collection or environment to a public workspace.
 * Make changes to a collection or environment that's present in a public workspace.
 * Write new documentation for a Postman Collection and make it public.
 * Make any changes to publicly available Postman documentation.
 
-Scan results are displayed in [Security audit reports](/docs/reports/security-audit-reports/) on the **Reports** section of the web dashboard.
+Postman delivers the scan results in the [Security audit reports](/docs/reports/security-audit-reports/) section of the [**Reports** dashboard](/docs/reports/reports-overview/).
+
+> **[Security Audit Reports are available on Postman Enterprise plans only.](https://www.postman.com/pricing/)**
```
Comment (claireriley) [inline src/pages/docs/api-security/token-scanner.md:53]: The only thing I'm wondering if you aren't on an Enterprise plan, how are you notified of a scan finding a token that's been made public? It says above that Token Scanner is available to all Postman plans, maybe we could mention how you'd be notified so folks understand results aren't just available to Enterprise teams

**Review by claireriley — APPROVED** [review]
Looks good! Just one question to clarify things for non-Enterprise folks

File: src/pages/docs/api-security/token-scanner.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/api-security/token-scanner.md:53]: ❗❗ 

Thanks for surfacing this question!

Comment (HeatherStenson) [reply src/pages/docs/api-security/token-scanner.md:53]: Per Suhas's answer in Jira, I'll merge this PR as-is and will plan on updating this when the planned product update launches.

**Pull request merged**